tests/SConscript

```diff
@@ -7,7 +7,7 @@ afwdataDir = env.ProductDir("afwdata")
 
 with open('test.txt', 'r') as f:
     tests = f.readlines()
-pybind11_ported_tests = [t for t in tests if not t.startswith('#')]
+pybind11_ported_tests = [t[6:].strip() for t in tests if not t.startswith('#')]
```

Comment:
Please add a comment to the SConscript file explaining the "6", and please consider using a named variable (in hopes of enhancing readability).